embulk-core/src/main/java/org/embulk/exec/GuessExecutor.java

```diff
@@ -56,8 +50,7 @@
     }
 
     public static void registerDefaultGuessPluginTo(Binder binder, PluginType type) {
-        Multibinder<PluginType> multibinder = Multibinder.newSetBinder(binder, PluginType.class, ForGuess.class);
-        multibinder.addBinding().toInstance(type);
+        throw new UnsupportedOperationException("GuessExecutor.registerDefaultGuessPluginTo no longer works.");
```

Comment:
extreme nit :) `no longer works` sounds like it's broken, more accurate wording could be better.

Reply:
Thanks! Agreed. Changed the wording to `is no longer available`.